github3/models.py

```diff
@@ -350,7 +350,7 @@ def _update_attributes(self, commit):
         #: SHA of this commit.
         self.sha = commit.get('sha')
         #: Commit message
-        self.message = commit.get('message')
+        self.message = commit.get('commit').get('message')
```

Comment:
So in this case, you want to do

```
commit.get('commit', {}).get('message')
```

Otherwise if `commit` isn't there you'll get an AttributeError for NoneType because `None` doesn't have a `.get` method.